### PR TITLE
Tweak token snapshot cards

### PR DESCRIPTION
**Pull request by buberdds**

### CODE REVIEW

File: src/app/pages/TokenDashboardPage/TokenSupplyCard.tsx [hunk 1]
```diff
@@ -32,7 +32,16 @@ export const TokenSupplyCard: FC = () => {
...
+              {!supplyString
+                ? t('common.missing')
+                : t('common.valuePair', {
+                    value: supplyString,
+                    formatParams: {
+                      value: {
+                        notation: 'compact',
+                      } satisfies Intl.NumberFormatOptions,
+                    },
+                  })}
```
Comment (csillag) [inline src/app/pages/TokenDashboardPage/TokenSupplyCard.tsx:44]: So this format is different than what we are trying to use in #703, although it's the very same data. I think we should use the same formatting everywhere...

File: src/app/pages/TokenDashboardPage/TokenTypeCard.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ export const TokenTypeCard: FC = () => {
                 color: COLORS.brandDark,
               }}
             >
-              {token?.type ? getTokenTypeName(t, token.type) : '-'}
+              {token?.type ? getTokenTypeStrictName(t, token.type) : '-'}
```
[the remark this answers is not in the document]
Comment (buberdds) [reply src/app/pages/TokenDashboardPage/TokenTypeCard.tsx:33]: Yep, waiting for Don feedback here. It looks pretty bad currently. 

Screenshot 2023-07-10 at 17 54 16

[the remark this answers is not in the document]
Comment (buberdds) [reply src/app/pages/TokenDashboardPage/TokenTypeCard.tsx:33]: Did he accept this earlier ?

[the remark this answers is not in the document]
Comment (csillag) [reply src/app/pages/TokenDashboardPage/TokenTypeCard.tsx:33]: But even if there is a problem with the formatting of the card, shouldn't we just fix the formatting issue? Why change the content?

[the remark this answers is not in the document]
Comment (csillag) [reply src/app/pages/TokenDashboardPage/TokenTypeCard.tsx:33]: > Did he accept this earlier ?

(We were also running slack chat as a side-channel, so yes, I think he really did.)

File: src/app/pages/TokenDashboardPage/TokenSupplyCard.tsx (hunk 1 above)
Comment (buberdds) [reply src/app/pages/TokenDashboardPage/TokenSupplyCard.tsx:44]: yeah I agree it should be the same. Will talk with Don tomorrow.

**Pull request closed without merging**